src/pudl/helpers.py

```diff
@@ -988,9 +1001,9 @@ def convert_cols_dtypes(df, data_source, name=None):
         zip_cols = [col for col in df.columns if 'zip_code' in col]
         for col in zip_cols:
             if '4' in col:
-                df[col] = zero_pad_zips(df[col], 4)
+                df.loc[:, col] = zero_pad_zips(df.loc[:, col], 4)
             else:
-                df[col] = zero_pad_zips(df[col], 5)
+                df.loc[:, col] = zero_pad_zips(df.loc[:, col], 5)
```

Comment:
Should/Can this fix be made in the `zero_pad_zips` function instead, so that it always applies, rather than just in this particular application? I'm pretty sure the function gets called elsewhere too in the ETL process doesn't it?

Reply:
these second loc's within the function call are not necessary. I was being over zealous here. I've removed them